app/Http/Controllers/UserController.php

```diff
@@ -17,21 +15,9 @@ public function get(User $user)
 
     public function store(UserStoreRequest $request)
     {
-        $inviteKeyValue = $request->invite_key;
-        $gameId = $request->game_id;
-
-        if (User::where('invite_key', $inviteKeyValue)->where('game_id', $gameId)->count() > 0) {
-            return CustomErrorService::failedApiResponse('Geen toestemming', [
-                'value' => ['De code is al in gebruik'],
-            ], 403);
-        }
-
         return User::create([
-            'username'   => $request->username,
-            'location'   => $request->location,
-            'invite_key' => $inviteKeyValue,
-            'game_id'    => $gameId,
-            'role'       => $request->role,
+            'username' => $request->username,
```

Comment:
Je vergeet nog de `InviteKey` te updaten. Nu kan je wel een gebruiker aanmaken, maar die wordt vervolgens niet gekoppeld aan een `InviteKey`, waardoor je nog steeds niet weet welke gebruikers in een spel zitten.

Reply:
![image](https://user-images.githubusercontent.com/44297604/115689511-6f9fe280-a35c-11eb-971a-6236c3331e00.png)
![image](https://user-images.githubusercontent.com/44297604/115689539-74fd2d00-a35c-11eb-8a2b-b08cc7d574f5.png)

Reply:
Zie snippets doorgestuurd door Leon en lees eventueel de Database sectie van de changelog. Daar wordt de hoe en waarom behandeld ^^.